app/views/posts/index.html.erb

```diff
@@ -1 +1,62 @@
-<h1>Here is a list of all the posts from this user</h1>
\ No newline at end of file
+<%= stylesheet_link_tag 'application', 'postindex', 'data-turbo-track': 'reload' %>
+<body>
+  <main class="flex">
+    <header></header>
+
+    <div class="profile">
+      <span class="profile-photo">
+        <img src="https://user-images.githubusercontent.com/37116322/222183427-3450696b-429a-4e9a-9e19-48bbe4b48edf.png" alt="profile" class="user-pic">
+      </span>
+      <span class="user">
+        <h2 class="name"><%= @user.name %></h2>
+        <span class="second-half">
+          <h5>#Number of posts:</h5>
+          <h6 class="numberofposts"><%= @user.posts_counter %></h6>
+        </span>
+      </span>
+    </div>
+
+    <% @posts.each do |post| %>
+      <div class="post">
+        <div class="post-body">
+          <h2 class="post-title"><%= post.title %></h2>
+          <p class="post-content">
+            <%= post.text %>
+          </p>
+          <div class="commentsandlikes">
+            <h5 class="sign">Comments:</h5>
+            <span class="numberofcomments"><%= post.comments_counter %></span>
+            <h5>Likes:</h5>
+            <span class="numberoflikes"><%= post.likes_counter %></span>
+          </div>
+        </div>
+
+        <ul class="comment-list">
+          <% post.comments.order(created_at: :desc).limit(5).each do |comment| %>
+          <li class="comment">
```

Comment:
- [ ] [OPTIONAL] While you implemented this which has the right outcome. The desired method to use here is the method used in the model to get the last 5 comments which may look like this:

```ruby
<% post.recent_comments do |comment| %>
```